Dockerfile.assisted-service-build

```diff
@@ -12,8 +12,8 @@ COPY --from=registry.ci.openshift.org/openshift/release:golang-1.16 /usr/bin/got
 COPY --from=registry.ci.openshift.org/openshift/release:golang-1.16 /usr/local/go /usr/local/go
 
 
-COPY ./hack/setup_env.sh ./dev-requirements.txt ./
-RUN ./setup_env.sh assisted_service
+COPY . .
+RUN ./hack/setup_env.sh assisted_service
```

Comment:
This makes it so that whenever you change a file in the working directory you have to re-run all of `setup_env.sh`. Currently it looks like this would increase _every_ skipper command by at least a few minutes.

Can you change this so that we first copy and run the script, then copy everything else so we can cache the script step? Ideally `COPY . .` should be as late in the Dockerfile as possible (you can also do the other COPY commands and maybe the `chmod` before it).